### PR TITLE
Added forms page and roster

### PR DESCRIPTION
**Pull request by KrispyKiwi315**
^

### CODE REVIEW

**Review by readerkid — APPROVED** [review]

File: src/content/pages/forms.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Forms"
+description: "this is meta description"
+draft: false
+---
+
+#### [Reimbursement Form](https://form.jotform.com/Troop150/troop-150-reimbursement-form)
```
Comment (txase) [inline src/content/pages/forms.md:7]: I think we should put this at the bottom of the list. I think the health forms are more commonly used by troop members.

File: src/content/pages/forms.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Reimbursements are event specific.  If you are looking to be reimbursed for multiple events, outings, advancement, or supplies, you need to submit separate reimbursement forms.
+
+No reimbursements will be paid or credited until the trip leader submits a complte "Event Recap Sheet". If you do not have a copy of the excel from, email Treasurer@Troop150.org for a copy.  Note if an expense is not listed on the form we will consider your expense a donation to the troop.
```
Comment (txase) [inline src/content/pages/forms.md:10]: Typo: "complte" -> "completed"

File: src/content/pages/forms.md (hunk 1 above)
Comment (txase) [inline src/content/pages/forms.md:7]: Let's figure out if there's a way to be more clear on what is a link (vs a section header).

**Review by txase — CHANGES_REQUESTED** [review]

**Review by txase — APPROVED** [review]
📋

**Pull request merged**